src/Greenshot.Editor/Drawing/Surface.cs

```diff
@@ -802,6 +805,15 @@ public IIconContainer AddIconContainer(string filename, int x, int y)
             return iconContainer;
         }
 
+        public IEmojiContainer AddEmojiContainer(string emoji, int x, int y, int size)
```

Comment:
Is there a default size, would that make sense? Just thinking out loud.

Reply:
There is one when you use the other constructor, this method is not used.

![image](https://user-images.githubusercontent.com/15875066/149653612-bc5df764-b770-4880-8d69-502423779650.png)

 I just mimicked what was done for the other containers. I can remove it.

Reply:
I meant to ask if there is a need for a default size for the method on the surface, not on the EmojiContainer.